File: frontend/src/app/_shared/models/pessoa/tecnico/tecnico.ts
import { Endereco } from "../endereco";
import { Pessoa } from "../pessoa";

export class Tecnico extends Pessoa {
    constructor(
        public matricula?: number,
        public ativo?: boolean,
        nome?: string,
        sexo?: string,
        idade?: number,
        endereco?: Endereco,
        dataNasc?: Date,
        cpf?: string,
        telefone?: string
    ) {
        super(nome, sexo, idade, endereco, dataNasc, cpf, telefone);
    }
}

File: frontend/mobile/app/src/main/java/com/tecno/aging/domain/models/pessoa/tecnico/Tecnico.kt
package com.tecno.aging.domain.models.pessoa.tecnico

import com.tecno.aging.domain.models.pessoa.Endereco

data class Tecnico(
    val id: Long = 0L,
    val cpf: String = "",
    val nome: String = "",
    val sexo: String = "",
    val dataNasc: String = "",
    val telefone: String = "",
    val matricula: String = "",
    val ativo: Boolean = false,
    val idade: Int = 0,
    val fotoPerfil: String? = null,
    val endereco: Endereco = Endereco(),
)